changelog/6251.feature.rst

```diff
@@ -0,0 +1 @@
+Add draw module which includes draw_equator and draw_prime_meridian functions to `sunpy.visualization`.
```

Comment:
```suggestion
Created the `sunpy.visualization.draw` module which includes `.draw_equator` and `.draw_prime_meridian` functions.
```
I forget if this will link correctly.

Reply:
```suggestion
Created the `sunpy.visualization.draw` module which includes :func:`~sunpy.visualization.draw_equator` and :func:`~sunpy.visualization.draw_prime_meridian` functions.
```